2022-participants/interested-organizations.md

```diff
@@ -10,7 +10,7 @@ Project | Link to project ideas page
 ------- | ---------------------------
 [SAMPLE] [The GloriousPickle Project](https://example.com) | [2022 PickleDoc Ideas](https://example.com)
 [cert-manager](https://cert-manager.io/) | [2022 cert-manager Ideas](https://cert-manager.io/docs/contributing/google-season-of-docs/2022/)
-[ExpressLRS](https://github.com/ExpressLRS) | [2022 ExpressLRS Ideas](https://www.expresslrs.org/2.0/gsod/)
+[Cilium](https://cilium.io/) | [2022 Cilium Idea](https://docs.cilium.io/en/latest/contributing/development/season_of_docs/)
```

Comment:
@xmulligan and @rominaminav You accidentally removed ExpressLRS from the list (added by @JyeSmith in #486).

Reply:
Thanks, see https://github.com/google/season-of-docs/pull/504